dev-util/gn/gn-01463.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A meta-build system that generates build files for Ninja"
+DESCRIPTION="GN is a meta-build system that generates build files for Ninja"
+HOMEPAGE="https://gn.googlesource.com/"
+COPYRIGHT="2008-2018 The Chromium Authors."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+srcGitRev="091169beda92aff21ded3e9dfb392ea12b10c0e0"
+SOURCE_URI="https://github.com/return/gn/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="e17c58a90e26212c74cd527b1a10d315ede15a64c9cf0755b4ef78418c379b76"
+SOURCE_DIR="gn-$srcGitRev"
+PATCHES="gn-01463.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	gn$secondaryArchSuffix = $portVersion
+	cmd:gn = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:clang
+	cmd:clang++
+	cmd:git
+	cmd:gcc$secondaryArchSuffix
+	cmd:g++$secondaryArchSuffix
+	cmd:ninja
+	cmd:python
```

Comment:
Is there any version requirement, or any python would do?

Reply:
@alaviss There isn't a specific version for python. It builds just fine with either Python 2 or 3.